src/struct_info.json

```diff
@@ -1824,18 +1844,30 @@
             ],
             "WGPUSurfaceDescriptorFromHTMLCanvasId": [
                 "nextInChain",
-                "sType",
                 "id"
             ],
+            "WGPUSurfaceDescriptorFromMetalLayer": [
+                "nextInChain",
+                "layer"
+            ],
+            "WGPUSurfaceDescriptorFromWindowsHWND": [
+                "nextInChain",
+                "hinstance",
+                "hwnd"
+            ],
+            "WGPUSurfaceDescriptorFromXlib": [
+                "nextInChain",
+                "display",
+                "window"
+            ],
```

Comment:
nit: These are unused on the Web, so don't need to be included.

Reply:
You mean the whole `WGPUSurfaceDescriptorFromMetalLayer`, -`WindowsHWND` and -`Xlib` structs?

Reply:
Yup